modules/claims_api/lib/bgs_service/manage_representative_service.rb

```diff
@@ -22,6 +22,12 @@ def read_poa_request(poa_codes: [])
             xml.SecondaryStatus status
           end
         end
+        if page_size || page_index
+          xml.send('data:POARequestParameter') do
+            xml.pageSize page_size if page_size
+            xml.pageIndex page_index if page_index
+          end
```

Comment:
cf. https://github.com/department-of-veterans-affairs/bgs-catalog/blob/4234350795ba516b5491d95bfd6582264fea782c/VDC/ManageRepresentativeService/ManageRepresentativePortBinding/readPOARequest/request.xml#L40-L43